### PR TITLE
11 BLB cards

### PR DESCRIPTION
**Pull request by Fulgur14**

### CODE REVIEW

File: forge-gui/res/cardsfolder/upcoming/caretaker's talent.txt
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/caretaker's talent.txt:1]: bad filename

File: forge-gui/res/cardsfolder/upcoming/caretaker's talent.txt [hunk 1]
```diff
@@ -0,0 +1,12 @@
+Name:Caretaker's Talent
+ManaCost:2 W
+Types:Enchantment Class
+T:Mode$ ChangesZoneAll | ValidCards$ Permanent.token | Destination$ Battlefield | TriggerZones$ Battlefield | ActivationLimit$ 1 | Execute$ TrigDraw | TriggerDescription$ Whenever one or more tokens you control enter, draw a card. This ability triggers only once each turn.
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/caretaker's talent.txt:4]: wrong valid

File: forge-gui/res/cardsfolder/upcoming/psychic_whorl.txt [hunk 2]
```diff
@@ -0,0 +1,7 @@
+Name:Psychic Whorl
+ManaCost:2 B
+Types:Sorcery
+A:SP$ Discard | ValidTgts$ Player | NumCards$ 2 | Mode$ TgtChoose | SubAbility$ DBSurveil | SpellDescription$ Target player discards two cards. Then if you control a Rat, surveil 2. (Look at the top two cards of your library, then put any number of them into your graveyard and the rest on top of your library in any order.)
+SVar:DBSurveil:DB$ Surveil | Defined$ You | Amount$ 2 | IsPresent$ Rat.YouCtrl
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/psychic_whorl.txt:5]: bad condition

File: forge-gui/res/cardsfolder/upcoming/caretaker's talent.txt
Comment (tool4ever) [reply forge-gui/res/cardsfolder/upcoming/caretaker's talent.txt:1]: still bad :p

Comment (Fulgur14) [reply forge-gui/res/cardsfolder/upcoming/caretaker's talent.txt:1]: "The Fulgur14-patch-71 branch of forge does not contain the path forge-gui/res/cardsfolder/upcoming/caretaker's talent.txt."

This is what it says, maybe it's just an artifact?

Comment (tool4ever) [reply forge-gui/res/cardsfolder/upcoming/caretaker's talent.txt:1]: no? it still has a space

**Review by tool4ever — APPROVED** [review]

**Pull request merged**